index.html

```diff
@@ -229,6 +233,31 @@ <h3 class="info" id="info">I'm always interested in hearing about new projects,
         <div class="hr1"></div>
         <div class="hr2"></div>
     </div>
+  </div>
+  <div class="popup">
```

Comment:
- [ ] Please make sure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). 
Let's compare Figma design and your design below 👇
## Desktop versions:
- Your popup here is not centered. Kindly fix it! 👇 
![Screenshot (92)](https://user-images.githubusercontent.com/67087664/147101265-5aae4c07-4f27-4dde-9b50-69fad942cb73.png)
- Figma design is centered👇 
![figma](https://user-images.githubusercontent.com/67087664/147101308-f266080c-f1aa-4484-91bb-aa45373d8cc0.png)


## Mobile versions:
- Your design here 👇 
![Screenshot (90)_LI](https://user-images.githubusercontent.com/67087664/147099430-000b387f-48a8-4c16-bbca-b4550972ec5b.jpg)

- Figma design 👇 
![figma](https://user-images.githubusercontent.com/67087664/147099263-85ecfb76-3599-4bc4-bbb1-695b126d84d9.jpg)